feder/assets/scss/style.scss

```diff
@@ -77,6 +77,43 @@ $gray: #F0F0F0;
   .page-title {
     color: $red;
   }
+  .email_metadata {
+
+  }
```

Comment:
@jacekll , a co to? 😉

Reply:
nazwijmy to.. artefaktem kompresji :) usunięte